client/components/NewProblem/NewProblem.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import './NewProblem.scss';
+
+export class NewProblem extends Component {
+    state = {
+        subject: '',
+        description: ''
+    };
+
+    setSubject = (event) => {
+        this.setState({subject: event.target.value});
+    };
+
+    setDescription = (event) => {
+        this.setState({description: event.target.value});
+    };
+
+    sendProblem = () => {
+        alert('subject: ' + this.state.subject + ', description: ' + this.state.description); //temp
```

Comment:
ALL user interactions in redux should dispatch action, maybe with some payload(in your example - payload is Problem object: {subject: '', description: ''});

never leave alerts, console.log's etc in src codebase